servicebus/data-plane/azure-servicebus/src/main/java/com/microsoft/azure/servicebus/management/TopicRuntimeInfoSerializer.java

```diff
@@ -104,6 +102,8 @@ private static TopicRuntimeInfo parseFromEntry(Node xEntry) {
                                                     case "TransferDeadLetterMessageCount":
                                                         topicRuntimeInfo.getMessageCountDetails().setTransferDeadLetterMessageCount(Long.parseLong(element.getFirstChild().getNodeValue()));
                                                         break;
+                                                    default:
```

Comment:
[SpotBugs-P2]: Switch statement found in com.microsoft.azure.servicebus.management.TopicRuntimeInfoSerializer.parseFromEntry(Node) where default case is missing